src/casync.c

```diff
@@ -48,6 +48,7 @@ typedef enum CaCacheState {
 struct CaSync {
         CaDirection direction;
         bool started;
+        uint64_t start_nsec;
```

Comment:
Note that we could get rid of `started`, and replace `if (s->started)` with `if (s->start_nsec != 0)`. Unless there are some speed considerations in using a bool instead of a uint64_t for this comparison, as it's called all the time in the main loop through the codepath  `ca_sync_step() -> ca_sync_start()`.

Reply:
Indeed. Maybe introduce a helper macro though to make this readable:

```
#define CA_SYNC_IS_STARTED(s) ((s)->start_nsec != 0)
```

I think this would keep the intention clearer than checking literally each time

Reply:
Done